src/Domain/Model/AddressType.php

```diff
@@ -0,0 +1,9 @@
+<?php
+declare( strict_types=1 );
+
+namespace WMDE\Fundraising\AddressChangeContext\Domain\Model;
+
+enum AddressType: string {
```

Comment:
@gbirke requested that the Donation address type enum not be string backed: https://github.com/wmde/fundraising-donations/pull/233

So I think we should do the same here to keep is consistent.

Reply:
@Sperling-0 as you were having questions about this earlier, the enum should should ideally look like the DonorType here, so you can remove the strings https://github.com/wmde/fundraising-donations/blob/3f103d1355d238da76a185d8d6572b45f795f001/src/Domain/Model/DonorType.php